zerver/lib/webhooks/git.py

```diff
@@ -17,6 +17,7 @@
 """
 
 FORCE_PUSH_COMMITS_MESSAGE_TEMPLATE = u"{user_name} [force pushed]({url}) to branch {branch_name}. Head is now {head}"
+CREATE_BRANCH_MESSAGE_TEMPLATE = '\n{user_name} created [{branch_name}]({url}) branch'
```

Comment:
I don't think template should start with '\n' it should be rather add by particular integration.
What if you'd like to just display info about creating branch?
It would look like:
```

user_name created ...
```